acoustid/future/fpindex/updater.py

```
@@ -0,0 +1,324 @@
+import array
+import asyncio
+import json
+import logging
+import signal
+from contextlib import AsyncExitStack
+from typing import Literal
```

Ruff (0.8.2)

`typing.Literal` imported but unused

Remove unused import: `typing.Literal`

(F401)